### PR TITLE
Gf/notification prefrences

### PR DESCRIPTION
**Pull request by George-field**
# [Notifications Preferences](https://app.shortcut.com/oazo-apps/story/4887/manage-notifications-ui-component-notifications-on-off-switch)

  
## Changes 👷‍♀️
- Added notifications preferences cards & section
  
## How to test 🧪
- Enable notifications by turning on the feature toggle in local storage. Click the notification Icon in the top right, then select the settings icon in the top right of the notifications modal.

### CODE REVIEW

File: components/notifications/NotificationPrefrenceCard.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { Box, Card, Flex, Text } from 'theme-ui'
+import { Toggle } from 'ui'
+
+interface NotificationPrefrenceCardPtops {
```
Comment (johnnieskywalker) [inline components/notifications/NotificationPrefrenceCard.tsx:5]: [minor][ a typo, `Props` is expected :D

**Review by johnnieskywalker — CHANGES_REQUESTED** [review]

File: components/notifications/NotificationPrefrenceCard.tsx (hunk 1 above)
Comment (George-field) [reply components/notifications/NotificationPrefrenceCard.tsx:5]: ah, dam it, thank you

File: ui/Toggle/Toggle.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        }}
+        type="checkbox"
+        onChange={() => setChecked(!checked)}
+      />
+      <Box
+        as="span"
+        sx={{
+          '&:before': {
+            position: 'absolute',
+            // TODO: Possibly create a function to do this so that is more readable
```
Comment (piekczyk) [inline ui/Toggle/Toggle.tsx:36]: I agree with TODO, if there is no other way to define these svgs we should definitely extract it to constant / function which returns it

File: components/notifications/NotificationsCenter.tsx [hunk 3]
```diff
@@ -52,7 +54,12 @@ export function NotificationsCenter({ isOpen }: { isOpen: boolean }) {
       <NotificationsCenterContent>
         <Grid sx={{ px: 2, mt: 3 }}>
           {showPrefrencesTab ? (
-            <p>Prefrences</p>
+            <Box>
```
Comment (piekczyk) [inline components/notifications/NotificationsCenter.tsx:57]: maybe it is worth to extract it as for example `NotificationPreferencesWrapper` even though it is simple box for now

**Review by piekczyk — CHANGES_REQUESTED** [review]

**Review by piekczyk — APPROVED** [review]

**Review by johnnieskywalker — APPROVED** [review]

**Pull request merged**